### ISSUE TITLE
Template is broken.

### ISSUE THREAD

**Issue by LeendertPlas** [mza921/Plex-Auto-Collections#176]
When running the default template, the alias Pixar is undefined. 
```
Traceback (most recent call last):
  File "C:\Plex\Plex-Auto-Collections-master\app\plex_auto_collections.py", line 1024, in <module>
    config = Config(config_path, headless=True)
  File "C:\Plex\Plex-Auto-Collections-master\app\config_tools.py", line 81, in __init__
    self.data = yaml.load(yml, Loader=yaml.FullLoader)
  File "C:\Python39-32\lib\site-packages\yaml\__init__.py", line 114, in load
    return loader.get_single_data()
  File "C:\Python39-32\lib\site-packages\yaml\constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "C:\Python39-32\lib\site-packages\yaml\composer.py", line 68, in compose_node
    raise ComposerError(None, None, "found undefined alias %r"
yaml.composer.ComposerError: found undefined alias 'Pixar'
  in "C:\Plex\Plex-Auto-Collections-master\config\config.yml", line 36, column 17
```

According to the readme it should be something like:

 ```
Pixar:
    plex_search:
      studio: Pixar
```
Instead of
```
  Pixar:
    studio: Pixar
```

**Comment by ramair02** [comment]
A bunch of `plex_search` stuff seems to be broken. Try using `tmdb_id`instead

**Comment by BlueSkillZ99** [comment]
Both expressions work just fine for me for movies and TV. Is it broken for you as well, @ramair02 ?

Can you post your full config (excl tokens) and output. There must be another issue

**Comment by ramair02** [comment]
TV is definitely broken for me. [172](https://github.com/mza921/Plex-Auto-Collections/issues/172)

And my understanding is that `plex_search` is somewhat broken overall. I think I read in another Issues thread that we were waiting on a fix for the plexAPI.

However, I haven't done a lot of testing with `plex_search`. I only use `decade: XXXX` in my config currently. The rest of it is `tmdb_id`, `tmdb_company`, `tmdb_actor`, `tmdb_director`, `tmdb_writer`, `tmdb_collection`, Trakt lists and IMDb lists.

To correct my earlier post, OP should try `tmdb_company: 3` for Pixar (not `tmdb_id`). I'll edit that post.
